bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/GmosSDtaXRuleTest.scala

```diff
@@ -58,9 +58,16 @@ class GmosSDtaXRuleTest extends AbstractRuleTest {
 
   private def expectSuccess(): Unit = {
 //    println(runCheck.mkString(",\n"))
-    assertTrue(runCheck.isEmpty)
+    assertTrue(! runCheck.exists { p =>
+      Set("GmosRule_X1", "GmosRule_Y2_4")(p.getId())
+    })
   }
```

Comment:
I actually implemented slightly different in order to resemble more the code already present, but the idea is the same.